task_2

```diff
@@ -0,0 +1,15 @@
+[dasha2@localhost ~]$ find /usr/share/man -type f -name "*help*"
+/usr/share/man/man1/help.1.gz
+/usr/share/man/man5/firewalld.helper.5.gz
+/usr/share/man/man8/mkhomedir_helper.8.gz
+/usr/share/man/man8/pwhistory_helper.8.gz
+/usr/share/man/man8/ssh-pkcs11-helper.8.gz
+[dasha2@localhost ~]$ find /usr/share/man -type f -name "conf*"
+/usr/share/man/man5/config.5ssl.gz
+/usr/share/man/man5/config-util.5.gz
+[dasha2@localhost ~]$ touch file1.txt
+[dasha2@localhost ~]$ ls
+file1.txt
+[dasha2@localhost ~]$ find . -empty -delete // this usage of command find without external -exec option deletes all empty files from current directory
```

Comment:
this usage of command without external -exec option deletes all empty files from current dir